pkg/controller/utils/errorscrub_test.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErrorScrubber(t *testing.T) {
+	cases := []struct {
+		name string
+
+		input    string
+		expected string
+	}{
+		{
+			name:     "aws request id",
+			input:    "failed to grant creds: error syncing creds in mint-mode: AWS Error: LimitExceeded - LimitExceeded: Cannot exceed quota for UsersPerAccount: 5000\n\tstatus code: 409, request id: 0604c1a4-0a68-4d1a-b8e6-cdcf68176d71",
```

Comment:
Is this input string realistic, or would it be
```suggestion
			input:    "failed to grant creds: error syncing creds in mint-mode: AWS Error: LimitExceeded - LimitExceeded: Cannot exceed quota for UsersPerAccount: 5000\n\tstatus code: 409\n\trequest id: 0604c1a4-0a68-4d1a-b8e6-cdcf68176d71",
```
? (see above comment)

Reply:
updated